### PR TITLE
Add eslint-config-react-native

### PR DESCRIPTION
**Pull request by FakeYou**
Since there wasn't any config for React Native I thought I would add one.

Based on [eslint-plugin-react-native](https://github.com/intellicode/eslint-plugin-react-native) and [eslint-plugin-react-native-a11y](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y).

### CODE REVIEW

File: packages/eslint-config-react-native/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-react-native",
+  "description": "STRV's ESLint config for React Native projects",
+  "version": "1.0.0",
```
Comment (robertrossmann) [inline packages/eslint-config-react-native/package.json]: Due to the way `lerna` cuts releases, please set this `version` here to `0.1.0`, and amend the commit introducing this package to contain this text in the commit's body:

```
feat: add @strvcom/eslint-config-react-native

BREAKING CHANGE: Initial release
```

That `BREAKING CHANGE` stanza is important. 😄 Thanks!

File: packages/eslint-config-react-native/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "license": "BSD-3-Clause",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/strvcom/code-quality-tools.git"
```
Comment (robertrossmann) [inline packages/eslint-config-react-native/package.json]: Not sure if this is important but in the other _package.json_ files we only have the `git://` protocol mentioned in the url. Don't know what's the difference, honestly. 🤷‍♂️

File: packages/eslint-config-react-native/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "peerDependencies": {
+    "eslint": "^5.3.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  }
+}
```
Comment (robertrossmann) [inline packages/eslint-config-react-native/package.json]: Please add `engines.node >=6` stanza to declare minimum Node.js runtime supported. See other _packge.json_ files for pointers. 🙏

File: packages/eslint-config-react-native/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Change Log
```
[the remark this answers is not in the document]
Comment (FakeYou) [reply packages/eslint-config-react-native/CHANGELOG.md]: Awesome, I was wondering about that

File: packages/eslint-config-react-native/accessibility.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
+/**
```
[the remark this answers is not in the document]
Comment (FakeYou) [reply packages/eslint-config-react-native/accessibility.js]: makes sense. I will merge it.

File: packages/eslint-config-react-native/index.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+ * @author      André Nanninga <andre@nannin.ga>
+ * @copyright   2019 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  extends: [
+    require.resolve('@strv/eslint-config-base'),
```
[the remark this answers is not in the document]
Comment (FakeYou) [reply packages/eslint-config-react-native/index.js]: I agree, I will extend the react ruleset.

**Comment by FakeYou** [comment]
I'm trying to remember if I used lerna to generate the package.json or I did in another way. Anyway I will update it and give you a ping when I have everything ready.

File: packages/eslint-config-react-native/readme.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# @strv/eslint-config-react-native
+
+> STRV's ESLint config for React-Native projects
+
+Suitable for linting React Native source files. This ruleset extends `@strv/eslint-config-react` but does not extend `@strv/eslint-config-react/optional` nor `@strv/eslint-config-react/style`.
```
Comment (robertrossmann) [inline packages/eslint-config-react-native/readme.md]: Could you please add a _style.js_ and _optional.js_ files which extend their respective rulesets from the base one?

Check the `eslint-config-node` directory, this ruleset does exactly that.

The reasoning behind this is that you should be able to use all three variants from each platform-specific ruleset without having to know that there is some "base" ruleset from which you might also want to extend. So in a project, you would just do:

```js
extends = {
  '@strv/react-native',
  '@strv/react-native/optional',
  '@strv/react-native/style',
}
```

and that's it. ❤️ 🚀

**Comment by robertrossmann** [comment]
I made some final cosmetic tweaks and fixed the _package.json_ and _lockfile_ - it's ready! 🎉 🚀 

Thank you @FakeYou for working on this for so long. I'll merge and publish now. 👍

**Pull request merged**